src/cli/commands/help.js

```diff
@@ -41,11 +43,14 @@ export function run(config: Config, reporter: Reporter, commander: Object, args:
     const getDocsLink = name => `${constants.YARN_DOCS}${name || ''}`;
     console.log('  Commands:\n');
     for (const name of Object.keys(commands).sort(sortAlpha)) {
-      if (commands[name].useless) {
+      if (commands[name].useless || unsupportedAliases[name] || Object.values(aliases).includes(name)) {
```

Comment:
Good catch. I think the proper support for Array.prototype.includes came with Node 6. I will update the PR.